src/output/icons.rs

```diff
@@ -120,6 +121,7 @@ const DIRECTORY_ICONS: Map<&'static str, char> = phf_map! {
     ".git"                => Icons::FOLDER_GIT,     // 
     ".github"             => Icons::FOLDER_GITHUB,  // 
     ".npm"                => Icons::FOLDER_NPM,     // 
+    ".pio"                => '\u{e682}',            // 
```

Comment:
Please make this a constant

Reply:
Actually just realized this is a directory.  We are trying to use only folder icons except for a few exceptions for things like Desktop, Photos, Downloads, ...